pushmsixscript/src/pushmsixscript/artifacts.py

```diff
@@ -0,0 +1,17 @@
+from scriptworker import artifacts
+from scriptworker.exceptions import TaskVerificationError
+from scriptworker.utils import get_single_item_from_sequence
+
+
+def get_msix_file_path(context):
+    artifacts_per_task_id, _ = artifacts.get_upstream_artifacts_full_paths_per_task_id(context)
```

Comment:
We don't have this in `scriptworker_client` yet. It's possible we don't need it, and it's possible we should port it over.